google/cloud/bigtable/bound_query.cc

```diff
@@ -36,9 +38,9 @@ InstanceResource const& BoundQuery::instance() const { return instance_; }
 google::bigtable::v2::ExecuteQueryRequest BoundQuery::ToRequestProto() const {
   google::bigtable::v2::ExecuteQueryRequest result;
   *result.mutable_instance_name() = instance_.FullName();
-  auto prepared_query = query_plan_->prepared_query();
-  if (prepared_query.ok()) {
-    *result.mutable_prepared_query() = query_plan_->prepared_query().value();
+  auto response = query_plan_->response();
+  if (response.ok()) {
+    *result.mutable_prepared_query() = sql_statement_->sql();
```

Comment:
`*result.mutable_query() = sql_statement_->sql();`

Don't try to populate google::bigtable::v2::ExecuteQueryRequest::prepared_query at this time. The logic in DataConnectionImpl::ExecuteQuery will do that.